lib/cobrato/resources/charging_type.rb

```diff
@@ -0,0 +1,21 @@
+module Cobrato
+  module Resources
+    class ChargingType < Base
+
+      def collection_name
+        'charging_types'
+      end
+
+      def list
+        http.get(resource_base_path) do |response|
+          bank_billets =  parsed_body(response)[collection_name]['bank_billets'].map do |item|
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.